src/components/SubmitResponsePage.jsx

```diff
@@ -47,8 +73,15 @@ const SubmitResponse = (props) => {
 										We're sorry, we encountered a problem processing your submission. We are working
 										to resolve this issue as quickly as possible.
 									</p>
+									<div className="cs-form-control">
+										<ButtonDisplay
```

Comment:
Since you are repeating this code, you could make a `HomeButton` component and import that into these modules where it's repeated.